web/src/reducers/working.js

```diff
@@ -72,13 +72,11 @@ const reducer = (state = initialState, { type }) => {
 const getAddAccountWorking = state => state.working.addAccount;
 const getEditAccountWorking = state => state.working.editAccount;
 const getEditOwnAccountWorking = state => state.working.editOwnAccount;
-const getSaveApdWorking = state => state.working.saveApd;
 
 export default reducer;
 
 export {
   getAddAccountWorking,
   getEditAccountWorking,
-  getEditOwnAccountWorking,
-  getSaveApdWorking
```

Comment:
Oh, nice catch! I think there's more in this file that we should remove too. There should be some maps that link action symbols to working state properties. I think the `SAVE_APD_*` actions are in those maps, and those can come out. There is a switch below that can be updated too.